task13/index.js

```diff
@@ -0,0 +1,92 @@
+const parent = document.getElementById("js-parent"); 
+
+const appendModalForParent = () => {
+    const modalPlace = document.createElement('section');
+    const modalText = document.createElement('p');
+    const closeBtn = document.createElement('input');
+    modalPlace.style.margin = '0 auto';
+    modalPlace.style.textAlign = "center";
+    modalPlace.style.width = '300px';
+    modalPlace.style.padding = '20px';
+    modalPlace.style.backgroundColor = "lightGray";
+    modalPlace.id = "modalPlace";
+    modalText.textContent = "クリックしてデータを取得します。";
+    closeBtn.style.margin = '0 auto';
+    closeBtn.style.display = "block";
+    closeBtn.id = "closeBtn";
+    closeBtn.type = "button";
+    closeBtn.value = "click";
+
+    parent.appendChild(modalPlace).insertBefore(modalText, modalPlace.firstChild).appendChild(closeBtn);
+};
```

Comment:
Wow, you made it in js!😃👍
I took the easy way out and wrote directly in HTML.😂

Reply:
I thought it should be made with JS!!
Should I recreate it in HTML?

Reply:
No, no, no! That's not true.
I just thought it was so great that you were so carefully participating in the assignment that I couldn't help but comment!👍